db/schema.rb

```diff
@@ -10,11 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_03_16_204230) do
+ActiveRecord::Schema.define(version: 2020_03_19_055350) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "linked_tokens", force: :cascade do |t|
+    t.string "token"
+    t.string "entity_type", null: false
+    t.bigint "entity_id", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["entity_type", "entity_id"], name: "index_linked_tokens_on_entity_type_and_entity_id"
```

Comment:
We should probably have an index on `token`.

Reply:
Good call. I made that change. Let me know if the added indexes make sense.